draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -99,7 +127,17 @@ const TooltipContent = ({ position, text, referenceElement, tooltipId }) => {
       role="tooltip"
       id={tooltipId}
     >
-      <div className={classnames(tooltipStyles.tooltipContent)}>{text}</div>
+      <div
+        className={classnames(tooltipStyles.tooltipContent, {
+          [tooltipStyles.default]: mood === "default",
+          [tooltipStyles.informative]: mood === "informative",
+          [tooltipStyles.positive]: mood === "positive",
+          [tooltipStyles.negative]: mood === "negative",
+          [tooltipStyles.cautionary]: mood === "cautionary",
```

Comment:
If you ensured the classname matches the value for `mood` then you could just use `tooltipStyles[mood]` instead of this object :)